notify/receivers.go

```diff
@@ -246,6 +247,23 @@ func (am *GrafanaAlertmanager) TestReceivers(ctx context.Context, c TestReceiver
 	return newTestReceiversResult(testAlert, append(invalid, results...), now), nil
 }
 
+func (am *GrafanaAlertmanager) buildSingleIntegration(r *GrafanaIntegrationConfig, tmpl *templates.Template) (*Integration, error) {
+	apiReceiver := &APIReceiver{
+		GrafanaIntegrations: GrafanaIntegrations{
+			Integrations: []*GrafanaIntegrationConfig{r},
+		},
+	}
+	integrations, err := am.buildReceiverIntegrationsFunc(apiReceiver, tmpl)
+	if err != nil {
+		return nil, err
+	}
+	if len(integrations) == 0 {
+		// This should not happen, but it is better to return some error rather than having a panic.
+		return nil, fmt.Errorf("failed to build integration")
```

Comment:
I think you want to use `errors.New` here as otherwise the linter will complain that you have not passed any arguments to the format string?

```suggestion
		return nil, errors.New("failed to build integration")
```